app/utils/search.py

```diff
@@ -122,6 +122,7 @@ def generate_response(self) -> str:
                                self.request_params,
                                self.config,
                                content_filter.near)
+        self.full_query = full_query
```

Comment:
This doesn't appear to get used anywhere

```suggestion
```

Reply:
It is used in the routes.py file when I call get_tab_contents [here](https://github.com/jacr13/whoogle-search/blob/7a86519b84bf3e232ae1268893e87e962e2b987a/app/routes.py#L263).